### ISSUE TITLE
Update to nemo 4.8.4 Error

### ISSUE THREAD

**Issue by emiliog50** [linuxmint/nemo#2686]
```
 * Nemo version (4.8.4)
 * windowed nemo
 * Linux Mint 20.1 Cinnamon
 * NVIDIA Corporation GK107 [GeForce GTX 650] *and* NVIDIA-DRIVER VERSION 450.102.04
 * 64 bit
 ```

**Issue**
The system requested update to nemo = 4.8.5 and nemo-python = 4.8.2 causing the error.

**Steps to reproduce**
Unexpected closings of the application and if there is more than one open and one of them closes, all of them are closed.

**Expected behaviour**
That does not close.

**Other information**
I hope to send the information correctly.
Here is the bug report I sent you https://github.com/linuxmint/nemo/issues/2685
Greetings.

[crash.tar.gz](https://github.com/linuxmint/nemo/files/6070296/crash.tar.gz)

**Comment by mtwebster** [comment]
Do you have nemo-terminal plugin enabled? If so, try disabling?

**Comment by emiliog50** [comment]
It was not activated.

**Comment by mtwebster** [comment]
Was it installed though?  Meaning: If you open edit->plugins, it's listed in the bottom section with a checkmark.

The reason I'm asking is I can reproduce warnings that are similar to what you provided using nemo-terminal (even if it's not currently visible).

**Comment by mtwebster** [comment]
Can you list what extensions you have enabled (from edit->plugins, bottom section).

**Comment by emiliog50** [comment]
![Captura de pantalla de 2021-03-02 18-25-37](https://user-images.githubusercontent.com/79927391/109688801-f8807600-7b84-11eb-80d0-7225b17a77fd.png)

**Comment by emiliog50** [comment]
![Captura de pantalla de 2021-03-02 18-28-48](https://user-images.githubusercontent.com/79927391/109689068-3a112100-7b85-11eb-9d63-a7877a490e08.png)

**Comment by emiliog50** [comment]
I went back to nemo 4.8.4 and python-nemo 4.6.0 and everything works fine.

**Comment by clefebvre** [comment]
Fixed in nemo 4.8.6
